Multiplayer/Action_Handlers.lua

```diff
@@ -46,14 +46,18 @@ end
 
 local function action_error(message)
 	sendDebugMessage(message)
-
 	Utils.overlay_message(message)
 end
 
 local function action_keep_alive()
 	ActionHandlers.Client.send("action:keepAliveAck")
 end
 
+local function action_disconnected()
+	Lobby.connected = false
+	Lobby.update_connection_status()
+end
```

Comment:
```suggestion
local function action_disconnected()
	Lobby.connected = false
	if Lobby.code then
		Lobby.code = nil
		G.FUNCS.go_to_menu()
	end
	Lobby.update_connection_status()
end
```

In case the user is in a lobby or game when they disconnect we should set the lobby code to nil and send them to the menu so they aren't stuck waiting for a server response while disconnected.

This should probably also create a modal/popup that says something along the lines of "Lost connection to the server" but I can implement that in a later PR.

Reply:
Good call. I thought about doing something like this but didn't know how. I added your suggestion, thanks.